rewrite-xml/src/main/java/org/openrewrite/xml/RemoveContentByPath.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.xml;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Option;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+
+/**
+ * Hypothetical + demonstration
+ * <p>
+ * todo
+ */
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class RemoveContentByPath extends Recipe {
```

Comment:
@aegershman can we call this `DeleteTag` to correspond to the yaml recipe `DeleteKey`?

Reply:
Updated, but am going to close this PR-- it was just a hypothetical put out there to externalize thoughts related to https://github.com/openrewrite/rewrite/issues/593 but there's no reason for this PR to hang out in the mean time 👍